src/components/notifications/YZJ.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+    <div class="mb-3">
+        <label for="yzjWebHookUrl" class="form-label">{{ $t("YZJ Webhook URL") }}<span style="color: red;"><sup>*</sup></span></label>
+        <input id="yzjWebHookUrl" v-model="$parent.notification.yzjWebHookUrl" type="text" class="form-control" required />
+        <label for="yzjType" class="form-label">{{ $t("YZJ Robot Type") }}<span style="color: red;"><sup>*</sup></span></label>
+        <input id="yzjType" v-model="$parent.notification.yzjType" type="text" class="form-control" required />
```

Comment:
What is this? is it maybe an enum (=> `<select>`)?

Could you please add a helptext what this is?